doc/Changelog.md

```diff
@@ -1,25 +1,27 @@
 Changelog {#Changelog}
 =========
 
-## git master {master)
+<!-- Uncomment at first change # git master {#master) -->
+
+# Release 1.6.0 (2015-10-04) {#Release010600}
```

Comment:
I picked the current date too, although this is the third way of formatting the date...

In the early projects I had 02-11-2015, in later ones 03-Nov-2015 to not confuse the Americans. I would vote for the latter format.